tutorials/mendix-salesorders/mendix-salesorders.md

```diff
@@ -1,27 +1,28 @@
 ---
-title: Create Sales Order App with SAP Cloud Platform Rapid Application Development by Mendix
+title: Create Sales Order App with SAP Rapid Application Development by Mendix
 description: Create a sales order app that consumes a service from the SAP Gateway Demo System (ES5).
 auto_validation: true
 primary_tag: products>sap-cloud-platform
+author_name: Paola Laufer
 tags: [  tutorial>beginner, topic>cloud, topic>odata, products>sap-cloud-platform  ]
 time: 20
 ---
 
 ## Prerequisites  
  - You are using a Windows desktop (or a Windows VM on a Mac).
- - You have access to an SAP Cloud Platform account. If not, you can open a trial account. See the [tutorial](https://developers.sap.com/tutorials/hcp-create-trial-account.html) or [documentation](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/65d74d39cb3a4bf8910cd36ec54d2b99.html).
- - You have completed the [Getting started with SAP Cloud Platform Rapid Application Development by Mendix](https://developers.sap.com/tutorials/mendix-onboarding.html) tutorial. Name your app **SAP Sales Orders**.
- - You have completed the [Create an account on the Gateway Demo system](https://developers.sap.com/tutorials/gateway-demo-signup.html) tutorial.
+ - You have access to an SAP BTP account. If not, you can open a trial account. See the [Get a Free Account on SAP BTP Trial] tutorial or the [documentation](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/65d74d39cb3a4bf8910cd36ec54d2b99.html).
```

Comment:
Please name the full product name at least once before using the abbreviation:
SAP Business Technology Platform

Reply:
Added Business Technology Platform